crates/torii/grpc/src/server/mod.rs

```diff
@@ -398,7 +400,7 @@ impl DojoWorld {
             .collect::<Vec<_>>()
             .join(" OR ");
 
-        if table == EVENT_MESSAGES_HISTORICAL_TABLE {
+        if table == EVENT_MESSAGES_HISTORICAL_TABLE || table == ENTITIES_HISTORICAL_TABLE {
             let count_query = format!(
```

Comment:
The execution of this query also fails if there's no data.
```
Error: Grpc(Status { code: Internal, message: "no rows returned by a query that expected to return at least one row"
```
Just by changing the `entity_updated_after` to some value to have a valid `where` clause.

Reply:
Thanks for the catch. Fixed!